programs/job/src/lib.rs

```diff
@@ -12,8 +11,12 @@ const GENERAL_SEED: &'static [u8] = b"general";
 pub mod job {
     use super::*;
 
-    pub fn initialize(ctx: Context<Initialize>, job_ad_id: String, max_amount_per_application: u32) -> Result<()> {
-
+    pub fn initialize(
+        ctx: Context<Initialize>,
+        job_ad_id: String,
+        max_amount_per_application: u32,
+    ) -> Result<()> {
+        assert_eq!(job_ad_id.len(), 36, "does not have a uuid length");
```

Comment:
added a check to ensure that we're passing something with the length of an uuid. Maybe it's pointless?

Reply:
No no, let it be